server/services/event-handlers.js

```diff
@@ -26,5 +26,13 @@ const registerEventHandlers = () => {
     subscribe("admin-verification", handleVerifyCaptureProcessed);
 }
 
-module.exports = registerEventHandlers;
+const handleEvent = (handleVerify) => (async (message) => {
+    handleVerify(message,()=>{
+        });
+});
+
+const dictEventHandlers = () => {

```

Comment:
We could just point dictEventHandlers to a dictionary directly instead of a function. Please see my code block above.